doc/source/whatsnew/v1.2.0.rst

```diff
@@ -505,6 +505,7 @@ I/O
 - Bug in :class:`HDFStore` was dropping timezone information when exporting :class:`Series` with ``datetime64[ns, tz]`` dtypes with a fixed HDF5 store (:issue:`20594`)
 - :func:`read_csv` was closing user-provided binary file handles when ``engine="c"`` and an ``encoding`` was requested (:issue:`36980`)
 - Bug in :meth:`DataFrame.to_hdf` was not dropping missing rows with ``dropna=True`` (:issue:`35719`)
+- :meth:`to_excel` supports fsspec resources (:issue:`33987`)
```

Comment:
can you add a little on what this means (e.g. you can now read/write to s3)

Reply:
it allows to write to s3 and so on (reading already worked) BUT `to_excel` does currently not support `storage_options` which might limit its usefulness. I'm happy to remove  this whatsnew entry and add it in a follow up that adds `storage_options`. It might be misleading to claim that to_excel supports fsspec but it doesn't support `storage_options`.

Reply:
Note that fsspec now has a configuration system, so you can define the default "storage options" even when you can't directly pass them. This is more of a workaround, though. I would say it supports fsspec as is, but adding storage_options would still be a good idea.